seqr/views/apis/anvil_workspace_api.py

```diff
@@ -1,48 +1,112 @@
 """APIs for management of projects related to AnVIL workspaces."""
 
 import logging
-from django.contrib.auth.decorators import login_required
+import json
 
+from django.shortcuts import redirect
+from django.contrib.sites.shortcuts import get_current_site
+
+from seqr.models import Project, CAN_EDIT
+from seqr.views.utils.json_to_orm_utils import create_model_from_json
 from seqr.views.utils.json_utils import create_json_response
-from settings import API_LOGIN_REQUIRED_URL
+from seqr.views.utils.file_utils import load_uploaded_file
+from seqr.views.utils.terra_api_utils import add_service_account
+from seqr.views.utils.pedigree_info_utils import parse_pedigree_table
+from seqr.views.apis.individual_api import add_or_update_individuals_and_families
+from seqr.utils.communication_utils import send_load_data_email
+from seqr.views.utils.permissions_utils import google_auth_required, check_workspace_perm
 
 logger = logging.getLogger(__name__)
 
 
-@login_required(login_url=API_LOGIN_REQUIRED_URL)
+def add_individuals_and_families(project, individual_records, user):
+    _, updated_individuals = add_or_update_individuals_and_families(
+        project, individual_records=individual_records, user=user
+    )
+
+    return '\n'.join(['Individual ID'] + [individual.individual_id for individual in updated_individuals])
```

Comment:
As I said before, this file shouldn't have a header

Reply:
Okay